rioxarray/xarray_plugin.py

```diff
@@ -0,0 +1,55 @@
+import os.path
+
+import xarray as xr
+
+from . import _io
+
+CAN_OPEN_EXTS = {
+    "asc",
+    "geotif",
+    "geotiff",
+    "img",
+    "j2k",
+    "jp2",
+    "jpg",
+    "jpeg",
+    "png",
+    "tif",
+    "tiff",
+    "vrt",
+}
+
+
+class RasterioBackend(xr.backends.common.BackendEntrypoint):
+    def open_dataset(
+        self,
+        filename_or_obj,
+        drop_variables=None,
+        mask_and_scale=True,
+        parse_coordinates=None,
+        lock=None,
+        masked=False,
+        variable=None,
+        group=None,
+        default_name="band_data",
```

Comment:
I saw you had `__band__` in `xarray-gdal`. Do you prefer that to `band_data`?

Reply:
Yes, this is user-visible in *rioxarray* while it was not in my toy plugin as I was also implementing the rewrite of the band names. I vote for `no underscores` for the users.